CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.11.6] - Unreleased
+### Changed
+- Improve behavior in AstartePairableDevice class when disconnection was received.
+
```

Comment:
No need for a new `0.11.6` entry given `0.11.5` was not released yet, just move the `Changed` section under the `0.11.5` entry